src/fosslight_dependency/package_manager/Pypi.py

```
@@ -202,20 +202,17 @@ def create_virtualenv(self):
         self.set_pip_activate_cmd(activate_cmd)
         self.set_pip_deactivate_cmd(deactivate_cmd)
 
-        env = os.environ.copy()
-        env['PYTHONUTF8'] = '1'
-
         cmd_list = [create_venv_cmd, activate_cmd, install_cmd, pip_upgrade_cmd, deactivate_cmd]
         cmd = cmd_separator.join(cmd_list)
 
         try:
-            cmd_ret = subprocess.run(cmd, shell=True, stderr=subprocess.PIPE, env=env)
+            cmd_ret = subprocess.run(cmd, shell=True, stderr=subprocess.PIPE)
```

Ruff (0.15.9)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)